customer-support/src/ChecklistTasks.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react';
+
+const notifyParent = (parent) => (message) => {
+  window.parent.postMessage(message, parent);
+}
+
+// We have to keep this task list up-to-date with the website's checklist
+const [SEND_MESSAGE] = [
+  'customer-send-message',
+];
+
+export const useChecklist = (customerClient, targetOrigin) => {
+  const notify = notifyParent(targetOrigin);
+  useEffect(() => {
+    const handleNewEvent = ({ type }) => {
+      switch(type) {
+        case 'message.new':
+          notify(SEND_MESSAGE);
+          break;
+        default:
+          break;
+      }
+    }
+    if (customerClient) {
+      customerClient.on(handleNewEvent);
+    }
+    return () => customerClient?.off(handleNewEvent);
+  })
```

Comment:
```suggestion
  useEffect(() => {
    const handleNewEvent = ({ type }) => {
      switch(type) {
        case 'message.new':
          notify(SEND_MESSAGE);
          break;
        default:
          break;
      }
    }
    if (customerClient) {
      customerClient.on(handleNewEvent);
    }
    return () => customerClient?.off(handleNewEvent);
  }, [customerClient])
```

Reply:
Thanks! Added in [438a9db](https://github.com/GetStream/website-react-examples/pull/7/commits/438a9dbb14bdee6073bb4d1ea9d185df4ca594b8)